packages/css/checkbox.css

```diff
@@ -53,115 +53,121 @@
     cursor: pointer;
   }
 
-  .description {
+  .fds-checkbox__description {
     margin-top: calc(var(--fds-spacing-3) * -1);
     color: var(--fds-semantic-text-neutral-subtle);
     grid-column: 2;
   }
 
-  .disabled > .input,
-  .disabled > .label,
-  .disabled > .input::before {
+  .fds-checkbox--disabled > .fds-checkbox__input,
+  .fds-checkbox--disabled > .fds-checkbox__label,
+  .fds-checkbox--disabled > .fds-checkbox__input::before {
```

Comment:
It was not possible to remove the direct child dependency here?

Reply:
Good observation, I think its just a bad habit leftover from some old stuff. 😅 

This can probably also be fixed in Radio and Switch also, as a lot of the code is the same there.